src/components/Chips/Chips.tsx

```diff
@@ -75,12 +76,21 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
     ref
   ) => {
     const overrideDataTestId = dataTestId || getTestId(ComponentDefaultTestId.CHIP, id);
+    const [hoverRef, isHovered] = useHover();
     const componentRef = useRef(null);
-    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef, hoverRef] });
 
     const backgroundColorStyle = useMemo(() => {
-      return { backgroundColor: disabled ? getCSSVar("disabled-background-color") : getElementColor(color, true) };
-    }, [disabled, color]);
+      let cssVar;
+      if (disabled) {
+        cssVar = getCSSVar("disabled-background-color");
+      } else if (isHovered) {
+        cssVar = getElementColor(color, true, true);
```

Comment:
not all colors having selected hover color. so I guess it will have a bad color for not supported colors. we need to add a check somehow if the color is not supported. I have an idea, let's talk about it in zoom?